arlo-client/src/components/CreateAudit.test.tsx

```diff
@@ -50,7 +50,8 @@ describe('CreateAudit', () => {
     })
   })
 
-  it('calls the /election/new endpoint for nonauthenticated user', async () => {
+  it.skip('calls the /election/new endpoint for nonauthenticated user', async () => {
+    // we have hidden the create new audit feature for unauthenticated users currently
```

Comment:
actually, I just moved it to it's own path: `/audit`

Reply:
@jonahkagan Se we should be testing that path separately, right? Or do you mean I should change the comment to be more clear?

Reply:
Actually, just saw that it's a completely different component that's on that path, so I updated the comment here accordingly.